releaseNotes/Microsoft.Featuremanagement.AspNetCore.md

```diff
@@ -0,0 +1,20 @@
+# Microsoft.FeatureManagement.AspNetCore
+
+[Source code][source_code] | [Package (NuGet)][package] | [Samples][samples] | [Product documentation][docs]
+
+## 4.0.0-preview2 - March 7, 2024
+
+### Enhancements
+
+* Added `TargetingHttpContextMiddleware`, which ensures TargetingContext is available for syncronous functions, like `TargetingTelemetryInitializer` in the `Microsoft.FeatureManagement.Telemetry.ApplicationInsights.AspNetCore` package. ([#409](https://github.com/microsoft/FeatureManagement-Dotnet/issues/409))
+* Added support for .NET 8 target framework. ([#364](https://github.com/microsoft/FeatureManagement-Dotnet/issues/364))
+
+### Breaking Changes
+
+No breaking changes in this release.
```

Comment:
How strongly do we feel the necessity of adding this no-breaking-change disclaimer? We don't do this in other release notes. Even if we put this, there is no guarantee of no breaking changes. There are just no _known_ breaking changes.

Reply:
I'll remove it for now- seeing as the guidelines don't suggest we have it.